sdk.go

```diff
@@ -195,3 +183,32 @@ func (v *Verifier) fetchJWKSFromRemote(ctx context.Context, u, keyID string) (*j
 	}
 	return &foundKey, nil
 }
+
+func (v *Verifier) getVerifyEndpoint(tok *jwt.JSONWebToken) (string, error) {
+	verifyEndpoint := v.staticJWKSEndpoint
+	if verifyEndpoint == "" {
+		out := jwt.Claims{}
+		if err := tok.UnsafeClaimsWithoutVerification(&out); err != nil {
+			return "", fmt.Errorf("couldn't get json web key: %w", err)
+		}
+
+		rawURL := out.Issuer
+		if !strings.HasPrefix(rawURL, "https://") && !strings.HasPrefix(rawURL, "http://") {
```

Comment:
we parse this URL just below anyway, you can compare a Scheme of URL object instead maybe?

Reply:
that's how it worked before. It ended up parsing `example.com:1234` as a url with a scheme of `example.com`.

Reply:
TIL